app/serializers/user_serializer.rb

```diff
@@ -1,5 +1,7 @@
 class UserSerializer < ActiveModel::Serializer
   has_many :affiliations, include: true, embed: :ids
+  has_many :events, include: true, embed: :ids, serializer: Notifications::ActivitySerializer
```

Comment:
Bad idea.  You don't want to serialize all "undismissed" notifications since the user last logged in or you'll be open for a world of hurt when the user goes on sabbatical for 3 months.